### ISSUE TITLE
Screen readers cannot report the active Paperback version number

### ISSUE THREAD

**Issue by amirsol81** [trypsynth/paperback#204]
This is not a major issue, but it is a bit strange that NVDA and JAWS cannot report the currently running version of Paperback. Normally, JAWS (Control+Insert+V) and NVDA (Shift+Insert+V) can announce the version of any application.
However, in Paperback, NVDA says "Unable to get version info," and JAWS simply says "Version" without providing any details.
